api/admin_test.go

```diff
@@ -301,6 +301,20 @@ func (ts *AdminTestSuite) TestAdminUserCreate() {
 				"isAuthenticated": false,
 			},
 		},
+		{
+			desc: "Ban created user",
+			params: map[string]interface{}{
+				"email":        "test4@example.com",
+				"phone":        "",
+				"password":     "test1",
+				"ban_duration": "24h",
```

Comment:
can you confirm the format for what is accepted by ban_duration and also document it in the README 👌